File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.ReferenceService.Models/hit-violation-type_item.ts


import { IHitViolationType_Item } from './ihit-violation-type_item';

export class HitViolationType_Item implements IHitViolationType_Item {
  
	public ViolationTypeID: number = 0;
	public Code: string = "";
	public Description: string = "";
	public IsActive: boolean = false;
  
}

File: GTTS/INL.ReferenceService/Models/HitViolationType_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.ReferenceService.Models
{
    public class HitViolationType_Item : IHitViolationType_Item
    {
        public int ViolationTypeID { get; set; }
        public string Code { get; set; }
        public string Description { get; set; }
        public bool IsActive { get; set; }
    }
}
